### DISCUSSION TITLE
Cloned repo into custom_modules but it does not show up in Comfy UI

### DISCUSSION THREAD

**Discussion by ddblue0** [crystian/ComfyUI-Crystools#106] [General]
Apologies if this is a newbie question, but I followed the "Install from Github" instructions to install this and it's like Comfy UI ignores it. There's no trace of it and crystal doesn't show up as a node in the Add Node menu. I'm using a "portable" version of Comfy if that matters.

**Comment by crystian** [comment]
shoud work ¯\_(ツ)_/¯, I'm not sure with portable version
